app/controllers/application_controller.rb

```diff
@@ -1,7 +1,2 @@
 class ApplicationController < ActionController::Base
-  before_action :identify_user
-
-  def identify_user
-    @user = cookies[:user_id] ||= SecureRandom.hex(10)
```

Comment:
We still need to create or find the current user - everything else should be based on this. What needs to be done:

Check if there exists a cookie, if so try to find the user by that cookie
If there exists no cookie/ user, create a new user and store his id in the cookie
Store the user as an instance variable (@user) so that it is available from all controllers
Make sure this is the only place where we create users, everywhere else should use this @user